components/resources/library/src/macosMain/kotlin/org/jetbrains/compose/resources/ResourceReader.macos.kt

```diff
@@ -5,6 +5,7 @@ import kotlinx.cinterop.usePinned
 import platform.Foundation.NSFileManager
 import platform.posix.memcpy
 
+@OptIn(ExperimentalResourceApi::class)
 actual suspend fun readResourceBytes(path: String): ByteArray {
```

Comment:
Do we need to mark these actuals with `@InternalResourceApi` as the `expect` is marked with it?